hermes-test-helper/src/main/java/pl/allegro/tech/hermes/test/helper/avro/AvroUser.java

```diff
@@ -16,29 +17,68 @@
     private static final String FAVORITE_COLOR_FIELD = "favoriteColor";
 
     private final Schema schema;
+    private final GenericRecord record;
 
-    public AvroUser() throws IOException {
-        this.schema = new Schema.Parser().parse(this.getClass().getResourceAsStream("/schema/user.avsc"));
+    public AvroUser() {
+        this(AvroUserSchemaLoader.load(), "", 0, "") ;
```

Comment:
maybe default value for **name** and **favouriteColor** should be something recognisable like "defaultName" or "defaultColor" instead of empty strings... easier to debug tests later...